### PR TITLE
Logical split of roles and user usage instructions

### PR DESCRIPTION
**Pull request by bravo-kernel**
Adds two new sections; Roles and Users with instructions per subject.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -63,10 +70,52 @@ return [
...
+- your database MUST have a ``roles`` table
+- your database MUST have a valid join table (e.g. ``roles_users``)
+- the configuration option ``multiRole`` MUST be set to ``true``
+
+Example of a record from a valid join table:
+
+```php
+'id' => 1
+'user_id' => 1
+'database_role_id' => 11
```
Comment (dereuromark) [inline docs/README.md]: why the database_ prefix?

File: docs/README.md [hunk 2]
```diff
@@ -63,10 +70,52 @@ return [
...
+
+Example of a record from a valid roles table:
+
+```php
+'id' => '11'
+'name' => 'User'
+'description' => 'Basic authenticated user'
+'alias' => 'user'
+'created' => '2010-01-07 03:36:33'
+'modified' => '2010-01-07 03:36:33
```
Comment (dereuromark) [inline docs/README.md]: missing `'` at the end

File: docs/README.md [hunk 3]
```diff
@@ -63,10 +70,52 @@ return [
...
+named ``roles``. If you prefer to use another table name simply specify it using the
+``rolesTable`` configuration option.
+
+>**Note:** make sure to add an "alias" field to your roles table (used as slug
+identifier in the acl.ini file)
+
+Example of a record from a valid roles table:
+
+```php
+'id' => '11'
```
Comment (dereuromark) [inline docs/README.md]: this can probably be just int 1 or sth

**Comment by bravo-kernel** [comment]
The prefix and ``11`` were copied from the tests, well spotted!

**Comment by bravo-kernel** [comment]
lgtm

**Pull request merged**